code/game/objects/items/devices/paicard.dm

```diff
@@ -157,6 +157,7 @@
 	audible_message("\The [src] plays a cheerful startup noise!")
 
 /obj/item/paicard/proc/alertUpdate()
+	flick("[initial(icon_state)]-alert", src)
```

Comment:
can you make the pAI card make a pinging noise or something as well (perhaps with a cooldown, in case ghosts try to spam it)?

Reply:
oh right, sure.